packages/rest-crud/README.md

```diff
@@ -15,11 +15,60 @@ npm install --save @loopback/rest-crud
 
 ## Basic use
 
-`@loopback/rest-crud` exposes two helper methods (`defineCrudRestController` and
-`defineCrudRepositoryClass`) for creating controllers and respositories using
-code.
+`@loopback/rest-crud` can be used along with `@loopback/model-api-builder` to
```

Comment:
I think this is not entirely correct. `@loopback/model-api-builder` provides only _Types and helpers for packages contributing Model API builders_ (quoting from README).

Did you perhaps mean the built-in [ModelApiBooter](https://github.com/strongloop/loopback-next/blob/master/packages/boot/src/booters/model-api.booter.ts)?

```suggestion
`@loopback/rest-crud` can be used along with the built-in `ModelApiBooter` to
```

Reply:
In this case, `CrudRestApiBuilder` implements `ModelApiBuilder` (not the booter). Quoting from the README: it also "provides a contract for extensions that contribute builders for repositories and controllers".